docker-compose.yml

```diff
@@ -183,6 +183,7 @@ services:
             - .docker/web/default.conf:/etc/nginx/conf.d/default.conf
             - public-api:/app/api/public
             - public-browser:/app/browser/public
```

Comment:
seems like this volume that comes from `browser_fpm` doesn't have an `assets/` in `public/` so no conflicts